content/intro-to-storybook/react/zh-TW/composite-component.md

```diff
@@ -1,19 +1,19 @@
 ---
-title: "組裝複合元件"
-tocTitle: "合成元件"
-description: "使用更簡單的元件 組裝複合元件"
+title: '組裝複合元件'
+tocTitle: '合成元件'
+description: '使用更簡單的元件 組裝複合元件'
 commit: '8db511e'
 ---
 
-上一章我們構建了第一個元件; 本章 我們學習 擴充套件構建TaskList的任務列表. 讓我們將 元件組合 在一起,看看在引入更多複雜性時會發生什麼.
+上一章我們構建了第一個元件; 本章 我們學習 擴充套件構建 TaskList 的任務列表. 讓我們將 元件組合 在一起,看看在引入更多複雜性時會發生什麼.
```

Comment:
@danielduan or @chinanf-boy -- pinging you here since you contributed the original Chineses translations. Prettier is formatting some of these files slightly differently. I recognize there are 2 Chinese translations and it appears to happen in both of them:

Seems like the common change is to insert a space around English words and Chinese characters. So:

`擴充套件構建TaskList的任務列表`

becomes

`擴充套件構建 TaskList 的任務列表`

The same happens in the `zh-CN` translation. Does this "break" the translation at all? Or is it an acceptable change? If it breaks things, I think we can probably ignore these files in Prettier.

Thanks 🙇